scripts/add_component_gate_trigger_job.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -xue
+
+cd "${REPO_DIR}"
+if [[ ! $(grep -s 'Component-Gate-Trigger_{repo_name}' "${PROJECTS_FILE}") ]] ; then
+  cat << EOF >> "${PROJECTS_FILE}"
+
+- project:
+    name: "${COMPONENT_NAME}"
+    repo_name: "${COMPONENT_NAME}"
+    repo_url: "${COMPONENT_REPO_URL}"
+
+    jobs:
+      - 'Component-Gate-Trigger_{repo_name}'
+EOF
+
+  issue_message="This issue was generated automatically as part of registering a new component."
+  issue_summary="Add new component gate trigger job for ${COMPONENT_NAME}"
+  issue=$(python ${WORKSPACE}/rpc-gating/scripts/jirautils.py \
+        --user "${JIRA_USER}" \
+        --password "${JIRA_PASS}" \
+        get_or_create_issue \
+          --project "${JIRA_PROJECT_KEY}" \
+          --summary "${issue_summary}" \
+          --description "${issue_message}" \
+          --label COMPONENT_GATE_TRIGGER \
+          --label "${COMPONENT_NAME}" \
```

Comment:
Does this trailing slash here cause any issues?

Reply:
It doesn't seem to [1], the issue gets created. It probably doesn't matter because it's wrapped in a `$()`.

[1] https://rpc.jenkins.cit.rackspace.net/job/Component-Release-Trigger/47/console